mkt/commonplace/templates/commonplace/index_react.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="{{ LANG }}" dir="{{ DIR }}">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no,
+                                   initial-scale=1, maximum-scale=1">
+    <title>Firefox Marketplace</title>
+
+    {% include "commonplace/includes/open_graph.html" %}
+
+    <meta name="theme-color" content="#fff">
+    <link rel="icon" href="{{ media('fireplace/img/logos/128.png') }}"
+          sizes="128x128">
+    <link rel="icon" href="{{ media('fireplace/img/logos/64.png') }}"
+          sizes="64x64">
+    <link rel="icon" href="{{ media('fireplace/img/logos/32.png') }}"
+          sizes="32x32">
+
+    {% if include_splash -%}
+      <link rel="stylesheet" href="{{ media(repo + '/css/splash.css') }}">
+    {% endif -%}
+    <link rel="stylesheet" href="{{ media(repo + '/css/bundle.css') }}">
+
+    {% if repo == 'fireplace' %}
+      <link rel="manifest" href="/manifests/standard.json">
+    {% endif %}
+
+    <link title="Firefox Apps"
+          rel="search" type="application/opensearchdescription+xml"
+          href="{{ url('opensearch') }}">
+    <meta name="fragment" content="!">
+
+    <link rel="dns-prefetch" href="//www.google-analytics.com">
+    <link rel="dns-prefetch" href="//stats.g.doubleclick.net">
+
+    {{ newrelic_header()|safe if repo == 'fireplace' }}
+  </head>
+  <body data-build-id-js="{{ BUILD_ID }}"
+        data-languages="{{ settings.AMO_LANGUAGES|json }}"
+        data-media="{{ MEDIA_URL }}"
+        {% if geoip_region %}data-region="{{ geoip_region.slug }}"{% endif %}
+        data-repo="{{ repo }}"
+        data-settings="{{ site_settings|json }}"
+        data-waffle-switches="{{ waffle_switches|json }}">
```

Comment:
Can we also move it to `<html>`? That we can can `React.render(<FluxComponent/>, document.body)` without nuking these.